tests/ui/conftest.py

```diff
@@ -14,31 +24,103 @@ def capabilities(capabilities):
     capabilities['marionette'] = True
     return capabilities
 
+
 @pytest.fixture
 def fxa_account(base_url):
-    url = DEV_URL if 'dev' in base_url else PROD_URL
+    url = DEV_URL if 'dev' or 'localhost' in base_url else PROD_URL
     return FxATestAccount(url)
 
 
-@pytest.fixture(scope='session')
-def jwt_issuer(base_url, variables):
+@pytest.fixture
+def jwt_issuer(base_url, json_file):
```

Comment:
I would create a `variables` fixture override that adds values from the JSON file to the existing `variables` dict.